linuxclientsetup/scripts/client-config

```diff
@@ -484,3 +484,6 @@ done
 
 #Write new variables to file
 updateVariables
+
+#Install dictionaries
+apt-get install -y myspell-en-gb
```

Comment:
This only belongs in the patch pushed via the web management, not in master. Put `myspell-en-gb` into `install/install.list`.